aerobites/cypress/e2e/main_page.cy.js

```diff
@@ -13,4 +13,77 @@ describe('Display headere on page load and drop down of airports', () => {
       .get('header').should('contain', 'AeroBites')
   })
 
-})
\ No newline at end of file
+  it('should display the "Show Favorites" button', () => {
+    cy.get('#show-favorites button').should('be.visible');
+  });
+
+//testing dropdown 
+it('should display a dropdown with airports', () => {
+  cy.get('.airports-select').should('be.visible');
+  cy.get('.airports-select').children('option').should('have.length', 7); // my mock data and the select airport dummy option
```

Comment:
Testing the first() and last() options can validate the data is populating in the order you expect it to, as well as validating some specific data points